include/swift/Basic/TaskQueue.h

```diff
@@ -81,13 +134,15 @@ class TaskQueue {
   /// \param Errors the errors from the task which finished execution, if
   /// available and SeparateErrors was true.  (This may not be available on all
   /// platforms.)
+  /// \param Process contains information like the operating process identifier
+  /// and resource usage if available
   /// \param Context the context which was passed when the task was added
   ///
   /// \returns true if further execution of tasks should stop,
   /// false if execution should continue
   using TaskFinishedCallback = std::function<TaskFinishedResponse(
       ProcessId Pid, int ReturnCode, StringRef Output, StringRef Errors,
-      void *Context)>;
+      TaskProcessInformation Process, void *Context)>;
```

Comment:
Minor preference for calling this variable ProcInfo or something; it's not a process in the sense of being a handle or something, just auxiliary information.